shotgun_api3/shotgun.py

```diff
@@ -2194,12 +2202,12 @@ def share_thumbnail(self, entities, thumbnail_path=None, source_entity=None,
 
         result = self._send_form(url, params)
 
-        if not six.ensure_text(result).startswith("1"):
+        if not result.startswith("1"):
```

Comment:
I'm assuming here that _send_form always returns a string, so we don't have to worry about calling
`startswith("1")` and `startswith(b"1")`?